### PR TITLE
Incorrect path in generating *.out tests command in integration tests documentation

### PR DESCRIPTION
**Pull request by k0T0z**
The path to ``bin/`` directory is written from the root as ``./bin/<godot_binary>`` however the path to ``scripts/`` directory is written from outside the root as ``godot-source/modules/gdscript/tests/scripts``, which is inconsistent I think.

### CODE REVIEW

File: contributing/development/core_and_modules/unit_testing.rst [hunk 1]
```diff
@@ -319,18 +319,24 @@ Therefore, the process of writing integration tests for GDScript is the followin
             if true # Missing colon here.
                 print("true")
 
-3. Generate ``*.out`` files to update the expected results from the output:
+3. Change directory to the root of godot source repository.
```
Comment (Calinou) [inline contributing/development/core_and_modules/unit_testing.rst:322]:
```suggestion
3. Change directory to the Godot source repository root.
```

File: contributing/development/core_and_modules/unit_testing.rst [hunk 2]
```diff
@@ -319,18 +319,24 @@ Therefore, the process of writing integration tests for GDScript is the followin
...
    .. code-block:: shell
 
-       ./bin/<godot_binary> --gdscript-generate-tests godot-source/modules/gdscript/tests/scripts
+       cd godot
+
+4. Generate ``*.out`` files to update the expected results from the output:
+
+   .. code-block:: shell
+
+       ./bin/<godot_binary> --gdscript-generate-tests modules/gdscript/tests/scripts
```
Comment (Calinou) [inline contributing/development/core_and_modules/unit_testing.rst:332]:
```suggestion
       bin/<godot_binary> --gdscript-generate-tests modules/gdscript/tests/scripts
```

**Comment by k0T0z** [comment]
Thank you!

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.1.

**Pull request merged**